<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FM1208-CPU_Qt_Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_beep">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>5</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>驱动读写器发出响声</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_getdevnum">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>5</y>
      <width>260</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读出设备全球唯一的设备编号，作为加密狗用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_request">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPU卡激活</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>681</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-3</number>
    </property>
    <property name="title">
     <string>功能二：初始化CPU卡</string>
    </property>
    <widget class="QCheckBox" name="checkBox_clear">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>21</y>
       <width>350</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择先将卡片清空再初始化      本次设定可应用的空间大小：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>56</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置卡片密码（16进制数）:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_maxsize">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>21</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cardkey">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>51</y>
       <width>270</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="text">
      <string>FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_init">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>初始化CPU卡</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>80</y>
       <width>491</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>卡密码与文件密码设任何关系，卡密码只是用来清空卡，读、写文件必须用文件密码来认证！</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>204</y>
      <width>681</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>功能三：创建文件和删除文件</string>
    </property>
    <widget class="QComboBox" name="comboBox_addfile">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>25</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>1号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6号文件</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>23</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>只读密码（16进制数）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_readkey">
     <property name="geometry">
      <rect>
       <x>241</x>
       <y>21</y>
       <width>270</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>52</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>读写密码（16进制数）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rwkey">
     <property name="geometry">
      <rect>
       <x>241</x>
       <y>50</y>
       <width>270</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>21</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>25</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件大小：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>71</y>
       <width>410</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>创建好的文件不能删除，要删除已有文件只能通过初始化功能清空所有文件!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>99</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1号文件 对应在目录 1680(十六进制) 下的 0001 文件，</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_addfile">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>71</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>创建标准文件</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2号文件对应在目录 1680 下的 002 文件 ，以此类推......</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_addeasyfile">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>103</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>创建易写文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>卡号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_uid">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>63</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>每次卡片重新进入感应区都要先激活卡片才能继续以下的操作。</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>681</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>功能四：修改文件密码</string>
    </property>
    <widget class="QComboBox" name="comboBox_changekeyfile">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>21</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>1号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6号文件</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>57</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件认证密码（16进制数）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_authkey">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>55</y>
       <width>270</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>86</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>新密码（16进制数）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_newkey">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>84</y>
       <width>270</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_authkey">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>用只读密码认证</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>用读写密码认证</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_changekey">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>修改只读密码</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>修改读写密码</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pb_changkey">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>修改文件密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>681</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>功能五：读、写文件信息</string>
    </property>
    <widget class="QComboBox" name="comboBox_rwfile">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>21</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>1号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5号文件</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6号文件</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>22</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件认证密码（16进制数）：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_authkey2">
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>19</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_authrwkey">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>用只读密码认证</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>用读写密码认证</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pb_read">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>读取文件数据</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>54</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>起始位置：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_rwbegin">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>83</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>读写长度：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_rwlen">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>读写数据：</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>591</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_write">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>写数据到标准文件内</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_easy_write">
     <property name="geometry">
      <rect>
       <x>539</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>写数据到易写文件内</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>41</y>
       <width>410</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>每次读卡最大长度为255字节，每次写卡最大长度247字节！</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_clear">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>673</y>
      <width>140</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>强制清空已初始化的卡</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_changecardkey">
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>672</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>更改卡片密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cardoldkey">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>668</y>
      <width>270</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cardnewkey">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>693</y>
      <width>270</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>672</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cpu卡16进制旧密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>697</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cpu卡16进制新密码：</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
